### DISCUSSION TITLE
Timeouts with big files

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/Oxalis-AS4#161] [Ideas]
Comment regarding handling of large file by Oxalis-AS4 : - 
".....Many experience this problem and it is expected to get worse once the new Peppol agreement framework actually (maybe) says something about the file sizes. It might be 100MB or something like that. And those files cause issues, even 30-60MB cause issues now.
It was reported that there once was a fix for this in Oxalis (OxalisCommunity/Oxalis-AS4#84) but that it was removed at some point. It was reported that re-implementing this fix makes it a lot faster. We should investigate if there is something to fix or not related to BounceCastle and the encryption/decryption speed.
Also it was discussed what is the default timeout values in Oxalis (is it 1 minute?) and is there a possibility to change default timeouts to 5 minutes in Oxalis? 5 minutes vs 1 minutes does help with the issue."

**Request to Oxalis Community Member**: If you see this as an issue in your day-to-day working, then please vote (thumbs up) for this issue.

**Comment by teedjay** [comment]
This happens regularly - not always related to Oxalis code, but infrastructure in front (lb's, fw's etc), config in haproxy/nginx type of routing or custom oxalis persistence storage code that uses databases with datatype restrictions on columns.

So one solution might be to have some "sane" defaults in Oxalis that we know will work for even large files and add guides on the WIKI pages here on GitHub with some tuning / config section for common setups?
